torch/csrc/jit/script/module.cpp

```diff
@@ -239,6 +239,25 @@ Module Module::clone_instance() const {
   return r;
 }
 
+Module Module::deepcopy() const {
```

Comment:
Module is a thin wrapper around Object. To preserve this property this function must be equivalent to `_ivalue()->deepcopy().`